Formula/avrdude.rb

```diff
@@ -30,46 +31,37 @@ class Avrdude < Formula
   end
 
   head do
-    url "https://svn.savannah.nongnu.org/svn/avrdude/trunk/avrdude"
+    url "https://github.com/avrdudes/avrdude.git", branch: "main"
 
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
+    depends_on "cmake" => :build
   end
 
-  depends_on "automake" => :build
   depends_on "hidapi"
-  depends_on "libftdi0"
-  depends_on "libhid"
-  depends_on "libusb-compat"
+  depends_on "libftdi"
+  depends_on "libusb"
 
-  uses_from_macos "bison"
-  uses_from_macos "flex"
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
 
   on_macos do
-    depends_on "libelf"
+    depends_on "libelf" => :build
```

Comment:
I think this only uses `libelf.a` (no dynamic lib in formula). Could also consider disabling ELF feature since not sure how to handle unmaintained `libelf`. Was previously looking at `elftoolchain` but not ideal to mix BSD license while `elfutils` doesn't build on macOS.